src/styles/themes/_definitions.scss

```diff
@@ -54,15 +54,15 @@ $color-tertiary-blue: var(--blue-color-60);
 $color-tertiary-violet: var(--violet-color-60);
 $color-tertiary-grey: var(--grey-color-60);
 
-$text-color-primary: var(--grey-color-90);
-$text-color-secondary: var(--grey-color-80);
+$text-color-primary: var(--text-primary-color);
+$text-color-secondary: var(--text-secondary-color);
```

Comment:
previous value of $text-color-secondary did not match that of --text-secondary-color... do we think this was intentional and I should undo this change or should we consider this a fix?

Reply:
Probably not a huge deal, only one implementation point in table.module.scss:
```
    &-filter-trigger {
     ...

      &:hover {
        color: $text-color-secondary;
        background: $table-header-filter-active-background-color;
      }
```